setLadokId/index.js

```diff
@@ -1,6 +1,31 @@
-function start () {
+require('dotenv').config()
+const canvas = require('@kth/canvas-api')(process.env.CANVAS_API_URL, process.env.CANVAS_API_TOKEN)
+const ldap = require('../lib/ldap')
+
+async function start () {
   console.log('Set LadokID (obtained from UG) to all users in a canvas course round (section)')
   console.log()
+
+  const sectionId = 'LT1016VT191'
+
+  await ldap.connect()
+
+  for await (const enrollment of canvas.list(`sections/sis_section_id:${sectionId}/enrollments`)) {
+    const kthId = enrollment.user.sis_user_id
+
+    if (kthId) {
+      const [user] = await ldap.search(`(ugKthId=${kthId})`)
+      const ladokId = user.ugLadok3StudentUid
+      console.log(kthId, ladokId)
+
+      await canvas.requestUrl(`/users/sis_user_id:${kthId}/custom_data/ladok_id`, 'PUT', {
+        ns: 'se.kth',
+        data: ladokId
```

Comment:
I think it's better to use a construct such as this:
```
'data': {
    'ladok_uid': msg.ladok3_student_uid
}
```
That means one can access it using a more "friendly url" (imo) such as `https://<canvas>/api/v1/users/<user_id>/custom_data/ladok_uid` - what do you think?

Reply:
The solution you are proposing is actually what the code does! :)

Reply:
Ah, yes, I see you've already baked it into the url!